definitions/pipelines/rnaseq.cwl

```diff
@@ -133,6 +136,7 @@ steps:
         run: ../tools/mark_duplicates_and_sort.cwl
         in:
             bam: merge/merged_bam
+            markdup_assume_sort_order: markdup_assume_sort_order
```

Comment:
Just hardcode this to the appropriate value for this pipeline, doesn't need to be an input.